packages/plugin-assets-retry/src/runtime/index.ts

```diff
@@ -99,10 +100,13 @@ function createElement(
     if (attributes.isAsync) {
       script.dataset.rsbuildAsync = '';
     }
-
     return {
       element: script,
-      str: `<script src="${attributes.url}" ${crossOriginAttr} ${retryTimesAttr} ${isAsyncAttr}></script>`,
+      str:
+        // biome-ignore lint/style/useTemplate: use "</" + "script>" instead of script tag to avoid syntax error when inlining in html
+        `<script src="${attributes.url}" ${crossOriginAttr} ${retryTimesAttr} ${isAsyncAttr}>` +
+        '</' +
+        'script>',
```

Comment:
can we just use `<\/script>` here

Reply:
😂 babel says no
<img width="1056" alt="image" src="https://github.com/web-infra-dev/rsbuild/assets/79413249/157766d1-4331-4df5-8678-c4780374ba71">

[babel-playground](https://babeljs.io/repl#?browsers=iOS%20%3E%3D%209%2C%20Android%20%3E%3D%204.4%2C%20last%202%20versions%2C%20%3E%200.2%25%2C%20not%20dead&build=&builtIns=false&corejs=false&spec=true&loose=true&code_lz=MYewdgzgLgBAHjAvDARAHgPQpgalRYAJwEsAHKAPmxhpgwxgE8BTCAKFElkaVTQB0MBEuSq1xNek1Ydw0GAC9eKQYNr9-KWVxgALXgAM0wspQFCipigboMwIIA&debug=false&forceAllTransforms=true&modules=commonjs&shippedProposals=true&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=false&presets=env%2Ctypescript&prettier=false&targets=&version=7.16.3&externalPlugins=&assumptions=%7B%7D)

Reply:
let's use string concat..